walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/BasicVerifiedIdStyle.kt

```diff
@@ -17,5 +20,5 @@ class BasicVerifiedIdStyle(
     val description: String,
 
     // Logo that should be displayed on the Verified Id.
-    val logo: Logo? = null
-): VerifiedIdStyle(name)
```

Comment:
why does name not have to be passed in anymore?

Reply:
I made VerifiedIdStyle and this class serializable. With that change it cannot have a parameter.